File: map/24iancsharp/ConsoleApplication2/ConsoleApplication2/Cheltuiala.cs
using System;

namespace ConsoleApplication2
{
    public class Cheltuiala
    {
        private TipCheltuiala tip;
        private float suma;
        private string desc;
        private string efecDe;

        public Cheltuiala(TipCheltuiala tip, float suma, string desc, string efecDe, DateTime data)
        {
            this.tip = tip;
            this.suma = suma;
            this.desc = desc;
            this.efecDe = efecDe;
            this.data = data;
        }

        public override string ToString()
        {
            return tip + ";" + suma + ";" + desc + ";" + efecDe + ";" + data;
        }

        private DateTime data;

        public TipCheltuiala Tip
        {
            get { return tip; }
            set { tip = value; }
        }

        public float Suma
        {
            get { return suma; }
            set { suma = value; }
        }

        public string Desc
        {
            get { return desc; }
            set { desc = value; }
        }

        public string EfecDe
        {
            get { return efecDe; }
            set { efecDe = value; }
        }

        public DateTime Data
        {
            get { return data; }
            set { data = value; }
        }
    }
}

File: map/practice_exam/src/Domain/Cheltuiala.java
package Domain;

import java.util.Date;

public class Cheltuiala {
    private TipCheltuiala tip;
    private float suma;
    private String desc;
    private String efectuataDe;
    private Date data;

    public TipCheltuiala getTip() {
        return tip;
    }

    public void setTip(TipCheltuiala tip) {
        this.tip = tip;
    }

    public float getSuma() {
        return suma;
    }

    public void setSuma(float suma) {
        this.suma = suma;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public String getEfectuataDe() {
        return efectuataDe;
    }

    public void setEfectuataDe(String efectuataDe) {
        this.efectuataDe = efectuataDe;
    }

    public Date getData() {
        return data;
    }

    public void setData(Date data) {
        this.data = data;
    }
}
